tests/bluetooth/shell/audio.conf

```diff
@@ -89,12 +89,12 @@ CONFIG_BT_ASCS_MAX_ASE_SNK_COUNT=2
 CONFIG_BT_PAC_SRC=y
 CONFIG_BT_ASCS_MAX_ASE_SRC_COUNT=2
 CONFIG_BT_BAP_BROADCAST_SOURCE=y
-CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=4
-CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT=4
+CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=1
+CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT=2
 
 CONFIG_BT_BAP_BROADCAST_SINK=y
-CONFIG_BT_BAP_BROADCAST_SNK_SUBGROUP_COUNT=2
-CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT=4
+CONFIG_BT_BAP_BROADCAST_SNK_SUBGROUP_COUNT=1
```

Comment:
This is limited by BT_BAP_BASS_MAX_SUBGROUPS - I am wondering if that one should be set to 1 as well?

Reply:
Probably. I think this value is being a few weird places, so might break a few things, but we'll see :)